src/pages/jobs.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import InPageNav from "@/components/InPageNav";
+
+const JobOpportunities = () => {
+  return (
+    <>
+      <div className="flex w-full flex-row justify-center bg-gray-100">
+        <div className="bg-background-white relative h-[1066px] w-[1463px] overflow-hidden">
```

Comment:
(nit) In general you tend to use a lot of custom values in tailwind - this is discouraged and it is recommended to use preset tailwind values where possible, with custom values being the exception. Also, some values correspond directly to existing tailwind presets, e.g. `tracking-[0]` => `tracking-normal`